.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+  "name": "Vite Development Container",
+  "image": "mcr.microsoft.com/devcontainers/base:jammy",
+  "features": {
+    "ghcr.io/devcontainers-extra/features/pnpm:2": {}
+  },
+  "postCreateCommand": "pnpm i && pnpm run build",
+
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  "forwardPorts": [
+    3000,
```

Comment:
Do we need to list 3000 and 8080 here? Shouldn't we list 4173 as well?

Reply:
Also, these should be on a single line to address the lint issue

Reply:
Done